docs/about/compatibility/openmp.md

```diff
@@ -472,3 +472,93 @@ A no-loop kernel is not generated if the OpenMP teams construct uses a `num_team
 #### Cross-team optimized reduction kernel generation
 
 If the OpenMP construct has a reduction clause, the compiler attempts to generate optimized code by utilizing efficient cross-team communication. New APIs for cross-team reduction are implemented in the device runtime and are automatically generated by clang.
+
+### Shared memory support
+
+Unlike other discrete GPU systems that have distinct CPU (system) and GPU memory, MI300A architecture features a single physical storage per socket which is the main memory for both CPU and GPU.
```

Comment:
This seems a little off, MI300A probably has several NUMA domains per socket. Calling a single physical storage per socket may be misleading.

Reply:
@ronlieb Could you please review this or add Carlo to respond to this?

Reply:
This is the compiler view of the architecture, not an accurate view of what happens in hardware, and essentially what the compiler designer consider a relevant architectural feature they need to optimize for.
Even if you have NUMA domains and multiple devices per socket, the thing remains that GPU and CPU threads do not have their own special memory where they should store their data to access it in the most efficient way. There is only a single physical storage that is the most efficient for them to access (save caches and registers). This model is what the compiler and runtime optimize for.